mne/viz/_brain/_brain.py

```diff
@@ -290,6 +290,10 @@ class Brain(object):
        and ``decimate`` (level of decimation between 0 and 1 or None) of the
        brain's silhouette to display. If True, the default values are used
        and if False, no silhouette will be displayed. Defaults to False.
+    theme : str | path-like
+        Can be "auto" (default), "light", or "dark" or a path-like to a
+        custom stylesheet. For Dark-Mode and automatic Dark-Mode-Detection,
+        `qdarkstyle` respectively `darkdetect` is required.
```

Comment:
CircleCI seems to have some issues with the new modules:

```
/home/circleci/project/mne/viz/_brain/_brain.py:docstring of mne.viz._brain._brain.Brain:97: WARNING: py:obj reference target not found: qdarkstyle
/home/circleci/project/mne/viz/_brain/_brain.py:docstring of mne.viz._brain._brain.Brain:97: WARNING: py:obj reference target not found: darkdetect
```

Probably the docstring needs to be updated:


```suggestion
        :mod:`qdarkstyle` respectively :mod:`darkdetect` is required.
```

And the list `intersphinx_mapping` in `doc/conf.py` too.